### PR TITLE
Update developer interview description

### PR DESCRIPTION
**Pull request by justinyaodu**

### CODE REVIEW

File: pages/students.tsx [hunk 1]
```diff
@@ -66,20 +66,49 @@ There are three stages for both designers and developers. We start with a resume
...
+
+- All of the topics for freshmen and sophomores, listed above
+- Designing JSON-compatible data representations in the language of your choice (don't worry about serialization/deserialization though)
+- Analyzing time and memory complexity
+- Data structures: stacks, queues, heaps, trees, and graphs
+- Algorithms: divide and conquer, recursion and backtracking, memoization, _basic_ dynamic programming
+
 ### How can I prepare for the technical components of the developer recruitment process?
 
-To prep for the systems design question:
```
Comment (anoushkadave) [inline pages/students.tsx:71]: why did we remove this? don't we want to share this info?

**Review by anoushkadave — CHANGES_REQUESTED** [review]
one question but otherwise lgtm, thank you!

File: pages/students.tsx (hunk 1 above)
Comment (justinyaodu) [reply pages/students.tsx:71]: we no longer have a systems design question in the phone screen!

**Review by anoushkadave — APPROVED** [review]
lgtm!

**Pull request merged**